src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml

```diff
@@ -64,6 +64,8 @@
             @Html.Gds().TextBoxFor(m => m.Organisation.Address.WebsiteAddress, new { @maxlength = CommonMaxFieldLengths.WebsiteAddress })
         </div>
 
+        <h2 class="govuk-heading-s">Enter the registered office or principal place of business address.</h2>
+        <p class="govuk-body">For registered companies this should be the registered office address.</p>
```

Comment:
@stevelee-civica  should the "For registered companies this should be the regist...." line be hint text? There is a hint text style

Reply:
updated